From 2a2c6863b81d79f2d0e977787c5581c851d3aa2a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Antoine=20Beaupr=C3=A9?= <anarcat@debian.org>
Date: Fri, 23 Mar 2018 16:35:19 +0000
Subject: [PATCH 1/2] add examples to manpage

I didn't get the syntax of the config file at first: I thought I could
just put `defaults` in there and it would do the right thing. Adding
examples should help others in figuring out the right way to do this.
---
 dh_sysuser | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/dh_sysuser b/dh_sysuser
index 62a9769..a9764f0 100755
--- a/dh_sysuser
+++ b/dh_sysuser
@@ -177,6 +177,16 @@ home directory will become empty, and user will be removed.
 
 =back
 
+=head1 EXAMPLES
+
+In F<debian/I<package>.F<sysuser>>, this will create a B<foo> user with
+defaults settings, will create a home in the default location for B<bar>,
+and a custom location for B<baz>:
+
+    foo defaults
+    bar home
+    baz home=/opt/baz
+
 =head1 SEE ALSO
 
 useradd(8)
-- 
2.14.2


From 2a5dd07e063723a1e5ebceed2e06e1a7fe73dc81 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Antoine=20Beaupr=C3=A9?= <anarcat@debian.org>
Date: Fri, 23 Mar 2018 12:39:22 -0400
Subject: [PATCH 2/2] clarify how to call dh_sysuser

At my first attempt, I just added --with=sysuser to the rules
target. That does not work: you need to specify arguments in the
config file or in the commandline as well.
---
 dh_sysuser | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/dh_sysuser b/dh_sysuser
index a9764f0..562a546 100755
--- a/dh_sysuser
+++ b/dh_sysuser
@@ -123,7 +123,10 @@ control over home directory permissions, you are welcome to file a bug.
 
 B<dh_sysuser> read its arguments from command line and file
 F<debian/I<package>.F<sysuser>> in pairs, first one being an username
-and second one is options. 
+and second one is options. The configuration file or commandline
+arguments must be used to create users: just calling `dh_sysuser`
+without arguments does nothing. Here are the options that can be
+specified after the username:
 
 =over
 
-- 
2.14.2

